src/Heading.js

```diff
@@ -0,0 +1,12 @@
AEoNe590GpWBBXoRTOZtEr3+ICWdXIHX2QKgE8oNV+OVdJEW39B3VWUi9He6UNwVlKiLVgZK6oXLibkDiXa72R2VR2Se9dD0HsK0POrSuGqkvMVxUQoNjAcrSlg9WFRANjLAmKVImMSl6QKYpVIIU5P+CiX8jjKBj8KiKzV3gAB3HsFFD/0L/oDnTHdcKP04t/M0/Va+reIbSgBvqsMkj0kOIjuBwrU7ihVDXU6lN4dIbtcMxyV5q8mfbR1emPwwKjSgELfrabJ9PHQziPnqs24s3CfldEM8X9IzxSQkHQrisUN5jlDLl0KiDsZ81XFdZlauWkemQTEzxzMhJV9YYyo5pcDAAgfiD8zM+0I2gbZ6VlZEbcLC0x5NNjjHqzxESZAjutOjRJQbQyTs0GXoCao6pHCll4fc8TvY35Tdt4YmZrUwB2krmlnxLTLrDkDUdecFoN8UVA0gGOIOVkV9Mos3AVXOMYgCJ91KNkdpJJDR1P+O6k5Y2rKKM7o0aupUqwmvuc4YbDiIHtHH2XlLpxE5x0WnWZTmGl0QOQJnqvQWvg1jqfmPe8S3cA0N4+oW9sMO3ewSi5Kj5vWqIJqe69TW8PlziACAHbSSCYkSJgLI1HR3UnFpHvj34XZDycctJkJ+NNbMao9BcStH+m9lw2hVvakR9TMtVLlpOsigutU3uTB6mZ7yql6ddbLjbSU3sQPWzPJXCU/UtYQTTCZZEK8bQsAeyu2i7sjIramFnNmUF9AttnIrKQCj66H/URwtcmZqKGw6FzzMoIuXOxJ5nnqm7QAGXAERwSlehlvo4J7KLQpPAAHpUU/Z+CnrPmlKze9lRzQIpNDqhB6EwD8rllbvc5u07SHNEuMAbjA+iBSrEAtk7TyOhI4xwt/wppL67qh81tJjWEuqOJLoH5GtBkyOkRhcsnxTbLrbPfeG/GFMOpWdU+Y8f2w6lSa1gI4Y31QR/zYGPiffmyBERjmMcr4TR05lU06DHsNTzXBtQU9rnA/gcWbQ/bjIMkc8L7Rpb3UWWtB7SDtLHl1Y1DLWkyH8kE8TEfZeT5UYxprs7cLb7A3OgUicmCeBjJ9l5W9tabHuZJ3Dc4t5IbBLY7zH0Tfi3xEaFwyXmDvbR/B6H7du7gkg7m/JHC+cap4jfUrUqrwA5tRr9gBbuZMmXg8EACI4VvGWaW70TyvGtUes1SzaIZ5lOQSXh7H8NbuMfGM+4XgX0AKjYcCAXOcWzkNzugjDT/AtJ9xUdQbPqNV7/7j2g/lbub5pO6fUCR7COqRtK1FlN4c0veXYcCGjY0OBbEzkkGewjrC7YOVbdnO1G9Kjf8ADuotcQH+ktBe0EuLnEl0wY7Ed17W1qDaCHbcB0EeqDkSOi8f4Y1p73SGUQ2mHQNtFhDnSKbi52X7RuEDoV7fQa9tcUzUY0uLnEVC4EE1Bg/4iOijmzOPaK48SfTBPuyDgz7yqPvHHkrUraXQjnb8ux+qX/8AiKfPm49oSR8nF/qGl4+T8f3Fad3CL/Wl0AnA4H7odTT2flqT89fsmKGnN/1HHwQqPNjav/Aqw5F/0f0ykHOBMQDP1XuTqQbTJJ4EYPVeEDdoEPHxlAv9TJG0HEQfdceSDzPRWlFfqPR2+r+bVbuw0Eux3IjKaurSjVBa8Y5EOI+/fqvHaZcw4E91u3WqNjHPH7qWbDxmuJXFJSi7D0vC1u5+N8dt3+U/Q8N2zT/+YPySVkW+rkRlNu1k91OazPXJmqPw1K2j20ZpM+wCyNR8O25aYAb1kGf3S1zqx7n3ysyvqB7rY8WS75M36V3swdVsWMkAmZPMRCxahhegv3B8rz91TiV7fjy1TODOt2hZ9RAc72VbmptE+/v8IDLiZHWYxPzldqpHI7COJVCVDVHUx9VDUGPf/KfkLTJKm4JajcbnGOAM8zM4/Yo0op2BoO2ojMrJKV2nUMxAxnn+dlnRlZsMqugYUSlK5MCf5+iikVs8FKIysREEiOIJHPKEooFhyjelrmPaXNe3h4cQfaI4XqrTx5VmiaznVfKa4eoAbpOA6OcRnuOq8SopzxQn+5WPHJKPRs6zqoqvLgTJcTkN4MRsMS3jhZj6m7nnjjogqJ4xUVSFbbdsbrXjnNY0uJDJ25MRMiB9/uhiuZJxJmZHU9UBRGgDFO5IjrHQ5EcxH0WnpHiWtbH+04AF257Yw7ER7D4hYiiVxT0wqTXQ/c6rVfzUfy4n1u6mYOcr1+j+M2eUynV3Bzdrd2XbhxJ7LwK6EJY4tdDLJJfT6Zq/iJtLYWOYfUN4DmklpMccj2I+q9FY6qxzWuaQ4ObuEgg7eJg5C+IymaOoVW5bUePhxUp+OpKh45mmfZK17APSf2SBrEnuvl79YrkyatQn/qXaesV28Vag/wC5GOHitMDycns+vWbHduPcK9eqes8r5KzxHdDivU4jkcIg8VXf/GP1aw/4UngyN3aK+3HVJM+qMuEf+pxyvkrvFd2RHmxmZDWA/cDhV/8AtF1j+6cOL+G5JiQe49vcrPx5MHsij6y+ulalZfM6vi26dPrAkgiKbMR0BiY+Us7xBcER5rud0iAZiOeyy8eS+meWJ9Fq3kP2mAC2WncJJ6iPhJG5D2F5xlwz2a4tk/Mfqvntxf1Hu3Pe4nvKqLx+fW7JkjceVeOOiTnZu3+qxgY54g/B9v8A2sk3jtxJOecQRujB+EoahMZ44+uVxrCVVWSpDdxdknrODOftHZdqXj9sHdOCDOOs4S49snuu7T3R2DQzY6h5c4mYPx3Wg3VaZMZHueFhkDqIXHMKKk0ZxTPQ/wBfTyA8TE84+6lO4BdO7oOojr/uF5tRNzYPWj1vnjuPuovJKLcwes4uriimUIooosYiiiixiKKKLGIooosYi6uKLGOqKKIoxFFFEV0Yi6VFErGRAooosA4oFFFmAi4oosY6ER/MLqiIGGYMKwUURFZVwQSY4UUWMFawdlHUh2UUSsJQ0wuKKIBP/9k=" alt = "logo"></img>
```

Comment:
Hmm I'm not sure this is quite right. The `src` of an `<img>` needs to be a URL of the image. So something like: `https://images.unsplash.com/photo-1492455417212-e162ed4446e1`

Reply:
This has now been corrected.